bzt/modules/python.py

```diff
@@ -619,6 +619,11 @@ def gen_action(self, action_config, indent=None):
         elif atype == "run" and tag == "script":
             action_elements.append(self.gen_statement('self.driver.execute_script(_tpl.apply("%s"))' %
                                                       selector, indent=indent))
+        elif atype == 'go':
+            if len(selector) > 0 and not param or len(param) == 0:
```

Comment:
1. selector: '' and param: None cause Exception (just check `go("")` action). You should use parenthesis (for param checks) to specify order of operations.
2. It isn't necessary to check len for empty value ('not value' is enough for both None and empty)